homeassistant/components/motionblinds_ble/strings.json

```diff
@@ -33,5 +33,32 @@
         "vertical": "Vertical blind"
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "battery": {
+        "name": "Battery"
+      },
+      "connection": {
+        "name": "Connected",
+        "state": {
+          "connected": "Yes",
+          "disconnected": "No",
+          "connecting": "Connecting...",
+          "disconnecting": "Disconnecting..."
+        }
+      },
+      "calibration": {
+        "name": "Calibrated",
+        "state": {
+          "calibrated": "Yes",
+          "uncalibrated": "No",
+          "calibrating": "Calibrating..."
+        }
+      },
```

Comment:
I think these entities could be more descriptive in their value.

Name: connectivity
connected
disconnected
connecting
disconnecting

Reply:
Personally I think "Connectivity: Connected" is a bit too verbose. The majority of times, currently it will say "Connected: Yes" or "Connected: No" which I think is nicer. "Connecting" and "Disconnecting" are kind of transitioning states to provide the user with a bit more information on what is happening

Reply:
That's why I opted for "Yes"/"No", I think it's easier to read

Reply:
Don't you use the key for automation and not the translation? For the majority of users I personally think this is the best option. "Connected: Yes" in my opinion is much easier to process (and not to mention translate to other languages) than something like "Connectivity: Connected"

```
        "state": {
          "connected": "Yes",
          "disconnected": "No",
          "connecting": "Connecting...",
          "disconnecting": "Disconnecting..."
        }
```

Reply:
For this one I'm looking to the connectivity device class, which is used in the binary sensors and looks like connectivity - connected - disconnected. Even tho this provides own translations I still think they should be in line with the translations other integrations use

Reply:
What about:
```
      "connection": {
        "name": "Connection",
        "state": {
          "connected": "Connected",
          "disconnected": "Disconnected",
          "connecting": "Connecting",
          "disconnecting": "Disconnecting"
        }
      },
```

But as you know this may result in longer translations in other languages, where in Dutch you would probably get "Verbinding: Verbonden", as opposed to "Verbonden: Ja". I still prefer the latter but if you want the former I will change it

Reply:
You could do Connection status

Reply:
Fixed it, please check

Reply:
I hope "Connection status: Disconnecting..." fits in the sensor entity?